### DISCUSSION TITLE
Spilo Image not published

### DISCUSSION THREAD

**Discussion by ibotty** [hydradatabase/hydra#161] [General]
Hi!

It would be great to have a hydra image based on spilo available.  This makes it possible to use zalando's postgres-operator.

**Comment by wuputah** [comment]
We switched from `spilo` to `postgres` for the main open source image last year by user request: https://github.com/hydradatabase/hydra/issues/22

We continue to use spilo internally, but we do not plan to publish a spilo-based image as a publicly supported image at this time. The build scripts do remain in the repo and do work if you wish to build it yourself.
